editor/store/reducer.js

```diff
@@ -1042,8 +1054,19 @@ export const blockListSettings = ( state = {}, action ) => {
 	return state;
 };
 
+export const autosave = ( state = null, action ) => {
```

Comment:
Can we add JSDoc describing what state is being tracked here.

Reply:
added in https://github.com/WordPress/gutenberg/pull/6257/commits/dfe6bc3f235ec86d7764a5ebfeb54445b878276d